### PR TITLE
Add support for generated constants in RPG:: classes

### PR DESCRIPTION
**Pull request by fmatthew5876**
Towards fixing https://github.com/EasyRPG/Player/issues/1516

Also adds panning constants, so that the default pan values don't need to be duplicated between player and liblcf.

### CODE REVIEW

**Comment by carstene1ns** [comment]
Thanks! This will also be very useful for #245.

**Comment by Ghabry** [comment]
Lets make a quick style discussion:
I find Hungarian notation really ugly. Is that some common idiom in C++11 code to prefix constexpr-constants with a German "konstant"-k?

**Comment by Ghabry** [comment]
Ah I thought it is k because c is the prefix for classes.

I dislike using e for enum but for the k we did a quick poll and the result is:

We don't **k**are

File: src/generated/rpg_savepartylocation.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * liblcf is Free/Libre Open Source Software, released under the MIT License.
+ * For the full copyright and license information, please view the COPYING
+ * file that was distributed with this source code.
+ */
+
+// Headers
+#include "rpg_savepartylocation.h"
+constexpr int RPG::SavePartyLocation::kPanXDefault;
+constexpr int RPG::SavePartyLocation::kPanYDefault;
+
```
Comment (Ghabry) [inline src/generated/rpg_savepartylocation.cpp]: I would like an empty line after the include and these 3 empty lines at the end removed

**Comment by fmatthew5876** [comment]
Fixed the whitespace. I also redefined the panning constants as X * 256

**Pull request merged**